readme.md

```diff
@@ -0,0 +1,105 @@
+# Intro
+
+This is MVP for one of our client (UTSEUS) which helps teams on hackathons exchange skills among team members and pay for their help.  It has three modules: web client (React), server (NodeJS) and storage with some business logic on blockchain (ethereum). This doc descriebs how to setup environment and run server backed by blockchian. 
+
+## Install environment
+
+A whole environment is described in scripts of package.json. Almost all what you need is run this scripts from the server folder. 
+
+```
+npm run install
+npm run testrpc
+npm run truffle
+# run server API tests
+npm run test-mocha
+# run unit tests
+npm run test-jest
+```
+
+## Configure truffle.js
+
+Use one of accounts from output of testrpc to put into this truffle.js configuration 
+
+```
```

Comment:
current checkedin only has this:

```
module.exports = {
  // See <http://truffleframework.com/docs/advanced/configuration>
  // to customize your Truffle configuration!
};
```

Reply:
couldn't it be made to be closer to what you document below?

Reply:
you should use code sample from documentation to put it into this file and update account in the filed `from`